linkerd/cert/cert.go

```diff
@@ -93,6 +93,6 @@ func GetDefaultX509Cert(serialNumber uint64, k *ecdsa.PublicKey, validFrom *time
 }
 
 // GetWindow returns cert validity window based on the arguments given
-func GetWindow(t time.Time, lifetime, clockSkewAllowance time.Duration) (time.Time, time.Time) {
+func GetWindow(t time.Time, lifetime, clockSkewAllowance time.Duration) (t1, t2 time.Time) {
```

Comment:
Please check the CI feedback here.  `named return "t1" with type "time.Time" found (nonamedreturns)`